groclient/experimental_test.py

```diff
@@ -0,0 +1,54 @@
+try:
+    # Python 3.3+
+    from unittest.mock import patch
+except ImportError:
+    # Python 2.7
+    from mock import patch
+
+from datetime import date
+from unittest import TestCase
+
+from groclient import Experimental
+from groclient.mock_data import mock_v2_prime_data_request, mock_v2_prime_data_response
+
+MOCK_HOST = "pytest.groclient.url"
+MOCK_TOKEN = "pytest.groclient.token"
+
+class ExperimentalTests(TestCase):
+    def setUp(self):
+        self.client = Experimental(MOCK_HOST, MOCK_TOKEN)
+        self.assertTrue(isinstance(self.client, Experimental))
+
+    @patch("groclient.lib.get_data_points_v2_prime")
+    def test_get_data_points(self, mock_get_data_points):
+        mock_get_data_points.return_value = mock_v2_prime_data_response.copy()
+
+        res = self.client.get_data_points(**mock_v2_prime_data_request)
+
+        self.assertEqual(len(res), len(mock_v2_prime_data_response))
+        self.assertIn('data_points', res[0])
+        self.assertIn('series_description', res[0])
+        point = res[0]['data_points'][0]
+        self.assertTrue(isinstance(point["start_timestamp"], int))
+        self.assertTrue(isinstance(point["end_timestamp"], int))
+
+    @patch("groclient.lib.get_data_points_v2_prime")
+    def test_get_data_points_df(self, mock_get_data_points):
+        mock_get_data_points.return_value = mock_v2_prime_data_response.copy()
+        df = self.client.get_data_points_df(**mock_v2_prime_data_request)
+
+        self.assertEqual(df.iloc[0]["start_timestamp"], date(2023, 5, 1))
```

Comment:
Can use `pandas.testing` assert_frames_equal, and init expected df directly so that it's easier to read:
https://github.com/gro-intelligence/gro/blob/main/infrastructure/derived-compute/historical_stats/test_generate_10y_avg.py#L28

Reply:
+1

Reply:
ha that's a good idea